azext_iot/digitaltwins/providers/model.py

```diff
@@ -7,7 +7,8 @@
 import json
 from azext_iot.common.utility import process_json_arg, scantree, unpack_msrest_error
 from azext_iot.digitaltwins.providers.base import DigitalTwinsProvider
-from azext_iot.digitaltwins.providers import ErrorResponseException
+# from azext_iot.digitaltwins.providers import ErrorResponseException
```

Comment:
We should remove the commented out import if its no longer used/useful.

Reply:
I just wanted to double check on this - the ErrorResponseException that is commented out is the one imported from:
azext_iot.sdk.digitaltwins.controlplane.models
but the one that works (as in the correct ErrorResponseException type thrown when there is an error) is imported from:
azext_iot.sdk.digitaltwins.dataplane.models

So it is correct that models is using the dataplane not the controlplane, right?

Reply:
That is right, models should be using dataplane ErrorResponseException.